pkg/scaffold/v2/main.go

```diff
@@ -72,14 +72,28 @@ func init() {
 	// %s
 }
 
+// Persmissions to do leader election.
+// +kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups="",resources=configmaps/status,verbs=get;update;patch
```

Comment:
we should scaffold this as a separate static role -- it's always the same, and shouldn't be something the user is concerned about.

Reply:
Done